Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +61,20 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+
+            for (int i = 0; i < array.Length / 2; i++)
+            {
+                //This to stop IndexOutOfRangeException
+                if (i + 1 >= array.Length) break;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4904050)

Reply:
This not an issue. And during lesson it was agreed to be ok.